src/styles/mixins/_dso-variables.scss

```diff
@@ -78,7 +78,10 @@ $dso-button-link-size: $u4;
 
 $dso-button-border-radius: 4px;
 $dso-button-border-width: 1px;
+$dso-button-line-height: $u3;
 $dso-button-margin-bottom: 1px;
+$dso-button-horizontal-padding: #{$u2 - $dso-button-border-width};
+$dso-button-vertical-padding: #{$u1 * 1.5 - $dso-button-border-width};
```

Comment:
Zie https://www.sassmeister.com/gist/e29fd28fff0fddaee15a86c1e03fbf96 voor wanneer `#{` en `}` nodig is

Reply:
Dit blijkt dus echt heel erg belangrijk te zijn om goed te doen. Zie https://www.sassmeister.com/gist/c57bcb6dcab58ccdb2567977ec744d73

Reply:
Documentatie zegt het volgende:

> ⚠️ **Heads up!**
> It’s almost always a bad idea to use interpolation with numbers. Interpolation returns unquoted strings that can’t be used for any further math, and it avoids Sass’s built-in safeguards to ensure that units are used correctly.

https://sass-lang.com/documentation/interpolation